### ISSUE TITLE
How do i train?

### ISSUE THREAD

**Issue by watermellon2018** [google/mediapipe#3598]
I use pose_landmark_heavy. I converted model tflite to tensorflow (tflite -> onxx -> tensorflow). I got Saved_model (variables and .pb). I want to train it. But when i try to load model i get this:
```
path = '../models/saved_model'
# tensorflow.python.saved_model.load.Loader._recreate_base_user_object.<locals>._UserObject 
model = tf.saved_model.load(path) 
```
Also i try this way:
```
inference = model.signatures["serving_default"] 
```
This way work, but i dont understand how i train it. Example, how i will get trainable_wight for optimizer: grads = tape.gradient(loss_value, model.trainable_weights)
Thanks

**Comment by kuaashish** [comment]
Hi @watermellon2018, This issue is more related to the tensorflow. Could you please raise this issue into tensorflow [repo](https://github.com/tensorflow/tensorflow/issues). 
Thank you!
